src/Toolkit/A.php

```diff
@@ -467,13 +480,7 @@ public static function move(array $array, int $from, int $to): array
 	 */
 	public static function missing(array $array, array $required = []): array
 	{
-		$missing = [];
-		foreach ($required as $r) {
-			if (isset($array[$r]) === false) {
-				$missing[] = $r;
-			}
-		}
-		return $missing;
+		return array_values(array_diff($required, array_keys($array)));
```

Comment:
I think this is a breaking change. A value like `['cat' => null]` was treated as missing before, but isn't anymore with `array_keys()`. I guess the expected behavior depends on the use case, but we should document this.

Reply:
I've added it as a breaking change to the 3.9.0-rc.1 release notes.